inference-engine/tests/functional/plugin/plaidml/shared_tests_instances/single_layer_tests/space_to_batch.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "common_test_utils/test_constants.hpp"
+#include "single_layer_tests/space_to_batch.hpp"
+
+using LayerTestsDefinitions::SpaceToBatchLayerTest;
+using LayerTestsDefinitions::spaceToBatchParamsTuple;
+
+namespace {
+
+const std::vector<InferenceEngine::Precision> inputPrecisions = {
+    InferenceEngine::Precision::FP32,
+    InferenceEngine::Precision::I32,
+};
+
+const std::vector<std::vector<size_t>> blockShapes = {
+    {1, 2, 2, 1}
+};
+
+const std::vector<std::vector<size_t>> pads_begins = {
+    {0, 0, 0, 0}
+};
+
+const std::vector<std::vector<size_t>> pads_ends = {
+    {0, 0, 0, 0}
+};
```

Comment:
Add test cases with nonzero padding

Reply:
sure, thanks for your reminder,  already add more nonzero padding test.